deploy/charts/crate-operator/values.yaml

```diff
@@ -0,0 +1,68 @@
+---
+# Default values for crate-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# If the `crate-operator-crds` is already installed, its installation
+# can be disabled by changing this value to `false`.
+crate-operator-crds:
+  # Make this disabled if you don't want to install
+  # the CRD dependency automatically.
+  enabled: true
+
+replicaCount: 1
+
+image:
+  repository: crate/crate-operator
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+imagePullSecrets: []
+nameOverride: "crate-operator"
+fullnameOverride: "crate-operator"
+
+envVars:
+  - name: CRATEDB_OPERATOR_BOOTSTRAP_TIMEOUT
+    value: "1800"
+  - name: CRATEDB_OPERATOR_DEBUG_VOLUME_SIZE
+    value: "128GiB"
+  - name: CRATEDB_OPERATOR_DEBUG_VOLUME_STORAGE_CLASS
+    value: "default"
+  - name: CRATEDB_OPERATOR_JMX_EXPORTER_VERSION
+    value: "1.0.0"
+  - name: CRATEDB_OPERATOR_LOG_LEVEL
+    value: "INFO"
+  - name: CRATEDB_OPERATOR_ROLLING_RESTART_TIMEOUT
+    value: "3600"
+  - name: CRATEDB_OPERATOR_SCALING_TIMEOUT
+    value: "3600"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "crate-operator"
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+
+podAnnotations: {}
+
+resources: {}
```

Comment:
I guess it makes sense to set sensible defaults here, as _we_ should know.

Reply:
Agree, would be nice to set some sensible defaults.

Reply:
I agree with you but I'm not sure which one.

Reply:
Let's set requests/limits to `250m` CPU and `128Mi` - just tried it on dev and it seems ok.